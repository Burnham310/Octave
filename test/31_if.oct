main = Section ;; {
    if true then 5 else 0 
    if false then {1 3 7} else {2 5 4 1}
    if 3 > 1 then [1 3 5] else [2 4 6]
}
