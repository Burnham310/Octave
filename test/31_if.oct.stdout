[
{
  "freq": 195.99354553222656,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 146.82925415039062,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 174.6103973388672,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000011373491242306228,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0,
  "duration": 0,
  "gap": 0,
  "amp": 0,
  "instrument": "dummy"
}

]